core/src/test/java/org/cru/contentscoring/core/service/impl/ContentScoreUpdateServiceImplTest.java

```diff
@@ -166,10 +166,31 @@ public void testGetPageUrl() {
         String pagePath = "/content/test/us/en/page-path";
         Page page = mockPage(pagePath, pagePath, "https://page.com" + pagePath);
 
-        String pageUrl = updateService.getPageUrl(page);
+        String pageUrl = updateService.getPageUrl(page, false);
         assertThat(pageUrl, is(equalTo("https://page.com" + pagePath + ".html")));
     }
 
+    @Test
+    public void testSkipPageUrl() {
+        String vanityPath = "/vanity-url";
+        String pagePath = "/content/test/us/en/page-path";
+
+        Page page = mockPage(pagePath, vanityPath, "https://vanity.com/content/test/us/en/" + vanityPath);
+        when(page.getVanityUrl()).thenReturn(vanityPath);
+
+        String vanityUrl = updateService.getPageUrl(page, false);
+        assertThat(vanityUrl, is(nullValue()));
+    }
```

Comment:
I'm confused by the scenario this test represents. The PR says that you are trying to send "both" the vanity url and the full url, but in this scenario the real URL is not sent, right?

Reply:
You're right, I had it that way at first, but second guessed myself at some point. It should send real URL in this case.